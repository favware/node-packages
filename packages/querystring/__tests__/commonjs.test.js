// eslint-disable-next-line
const { stringify, parse } = require('../src');

describe('CommonJS usage', () => {
  describe('Stringify', () => {
    test('should work with empty input', () => {
      const expected = '';
      const actual = stringify({});
      expect(actual).toMatchSnapshot();
      expect(actual).toBe(expected);
    });
  });

  describe('Parse', () => {
    test('should work with empty input', () => {
      const expected = {};
      const actual = parse('');
      expect(actual).toMatchSnapshot();
      expect(actual).toMatchObject(expected);
    });
  });
});
