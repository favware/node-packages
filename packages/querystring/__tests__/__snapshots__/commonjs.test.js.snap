// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommonJS usage Parse should work with empty input 1`] = `Object {}`;

exports[`CommonJS usage Stringify should work with empty input 1`] = `""`;
